### ISSUE TITLE
File sorting and In-file finder

### ISSUE THREAD

**Issue by costin88boss** [Minehut/Meta#876]
### Feature request

**Checklist**
- [X] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [X] This is not a bug report, support request or plugin request
- [X] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [X] This issue does not contain any sensitive information 

**Platform**
Website

**Describe the feature**
If you're a server host managing the server files like players userdatas, plugin configurations and player data files,
or going through a plugin file just to find a certain line because there is no finder, then this feature will help you go through this much time loss.

Basically, you'll now have a search bar in the files, and a sorter in the directory so that you can sort A-Z, by data size, and by file type.

**How the feature is useful**
It will save so much time that people have wasted only trying to find a certain line in a file or a certain user's UUID (I had to find someone's UUID, and to find it i needed the size, because there are a bunch of new players that had under 1 KB)
---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**
